README.md

```diff
@@ -301,18 +310,15 @@ This step assumes that you have a suitable Nvidia-GPU with [Cuda v9.0](https://d
 Additional notes:
 * Options for `--model` are currently listed in `src/model_restore_helper.get_model_class_from_name`.
 
-* Hyperparameters are specific to the respective model/encoder classes; a simple trick to discover them is to kick off a run without specifying hyperparameter choices, as that will print a list of all used hyperparameters with their default values (in JSON format).
-
-* By default, models are saved in the `/resources/saved_models` folder of this repository, but this can be overridden as shown above.
```

Comment:
Why was the default save location removed?

Reply:
oh because this is mentioned in the previous paragraph already, a few lines above we say:
"By default models are saved in the resources/saved_models folder of this repository."